showcase/src/app/demo/pretty-print/demo-pretty-print.component.ts

```diff
@@ -28,6 +33,19 @@ export class PrettyPrintComponent implements OnInit {
 		this.addJson();
 		this.addXml();
 		this.addSql();
+
+		this.unformattedData = "";
+		this.selectedDataFormat = "";
+		this.highlightingEnabled = false;
+		this.fileTypes.forEach((fileType: FileType) => this.dataFormats.push(fileType.format));
```

Comment:
You can improve a bit this line by using the Array.map() function instead:

`this.dataFormats = this.fileTypes.map((fileType: FileType) => fileType.format);`